examples/thermostat/thermostat-common/src/thermostat-delegate-impl.cpp

```diff
@@ -347,52 +339,61 @@ CHIP_ERROR ThermostatDelegate::RemoveFromThermostatSuggestionsList(uint8_t uniqu
     return CHIP_NO_ERROR;
 }
 
-uint8_t ThermostatDelegate::GetUniqueID()
+bool ThermostatDelegate::FindInList(uint8_t uniqueIDToFind)
+{
+    for (size_t index = 0; index < static_cast<size_t>(mNextFreeIndexInThermostatSuggestionsList); index++)
+    {
+        if (uniqueIDToFind == mThermostatSuggestions[index].GetUniqueID())
+        {
+            return true;
+        }
+    }
+    return false;
+}
+
+CHIP_ERROR ThermostatDelegate::GetUniqueID(uint8_t & uniqueID)
 {
     uint8_t maxUniqueId = 0;
 
     for (size_t index = 0; index < static_cast<size_t>(mNextFreeIndexInThermostatSuggestionsList); index++)
```

Comment:
Again, might be clearer without the casting if we had a nice helper for the iteration.....  Nothing here actually _uses_ index for anything.

Reply:
done.